requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements-base.txt
+tox==4.4.12
+build==0.7.0
```

Comment:
Just curious why `build` is here and not in either `-base` or `-build`?

Reply:
Based on historic commits and comments...
* `requirements-build` is specifically for dependencies required for building installers
* `build ` was added in 2021 while `requirements-base` was added in 2022 and seems it was missed

I would agree that it makes sense to move `build` to `requirements-base`